script/setup/install-runc

```diff
@@ -7,7 +7,7 @@ set -eu -o pipefail
 
 RUNC_COMMIT=$(grep opencontainers/runc ${GOPATH}/src/github.com/containerd/containerd/vendor.conf | cut -d " " -f 2)
 
-go get -u github.com/opencontainers/runc
+go get -d github.com/opencontainers/runc
```

Comment:
why remove the request to update dependencies for runc ?

Reply:
`go get -u github.com/opencontainers/runc` will build and install runc to `$GOPATH/bin`, which is also in `$PATH`.

containerd will use that `runc` instead of the one we build with `make` and `make install`.